SPECS-EXTENDED/reaper/reaper.spec

```diff
@@ -0,0 +1,156 @@
+Summary: Cassandra Reaper
+Name: reaper
+Version: 3.1.1
+Release: 1%{?dist}
+License: Apache 2.0
+Vendor: Microsoft Corporation
+Distribution: Mariner
+Group: Applications/System
+
+URL: http://cassandra-reaper.io/
+Source0: https://github.com/thelastpickle/cassandra-reaper/archive/refs/tags/%{version}.tar.gz
+Source1: cassandra-reaper-%{version}.tar.gz
```

Comment:
What's the difference between `Source1` and `Source0`? Also, I don't see `Source0` in the signatures. Does this package build locally with signature verification enabled?

Reply:
It is a mistake. Source 1 is actually the file name in the last part of Source0. It will be removed.